frame/election-provider-multi-phase/Cargo.toml

```diff
@@ -70,12 +69,15 @@ std = [
 	"sp-arithmetic/std",
 	"frame-election-provider-support/std",
 	"log/std",
+
```

Comment:
The new lines are weird here.

And doesn't the enabling of features of optional deps, currently mean that the dep is activated?

Reply:
Aka I would not add these features here.

Reply:
I thought a new line was warranted because those is a "separate set" as in: These are the benchmarking deps. If I remove the new line I would also remove the other ones, right?

> And doesn't the enabling of features of optional deps, currently mean that the dep is activated?

Unfortunately, yes. Cargo is dumb. But if we don't do it it breaks the build. This is why I noticed it. If you go to this directory and run `cargo {check, build}  --features runtime-benchmarks` it will have a build error without proper std propagation.

Reply:
Yeah, I assumed something like this.. Let's keep it. As long as it compiles, it should be fine :P